### ISSUE TITLE
Load timeout with RequireJS

### ISSUE THREAD

**Issue by yanickrochon** [corejavascript/typeahead.js#66]
I just replaced "twiter/typeahead.js" with this repository this morning, hoping to resolve an issue. But, instead, it won't load anymore.

I installed the component using **bower** (i.e. it changed `"typeahead.js": "^0.11.1"` to `"typeahead.js": "corejs-typeahead#^0.11.1"` in my `bower.json` file).
I reviewed my build to make sure the exact same file (i.e. `typeahead.bundle.min.js`) is referenced in my dependencies (all seems to be unchanged). Launched the app, and I get RequireJS to throw `"Uncaught Error: Load timeout for modules: typeahead"`.

My RequireJS config is declared as such :

```
require.config({
  ...
  paths: {
    ...
    'typeahead': 'path/to/typeahead.bundle.min',
    'bloodhound': 'path/to/typeahead.bundle.min',
    ...
  }
});
```

Has this happened to anyone? How to resolve this?

**Comment by yanickrochon** [comment]
Ah! Nevermind, I forgot I had changed the `defined` call from `typeahead.js` to `typeahead`.

This should be something to change, as noted in some previous issues....

**Comment by yanickrochon** [comment]
Using the most recent JS, in this repo's `dist` folder, I see that `define` does not declare anything anymore. This breaks my `require.config`.

**Comment by yanickrochon** [comment]
Please, fix this before releasing `0.11.2` (See issue #55)

**Comment by skovmand** [comment]
+1

**Comment by elgervb** [comment]
+1 I'm also getting errors whit requireJS. Please rename the definition from **typeahead.js** to **typeahead**

I've fixed this locally by copying the 0.11.1 library and changed
```
 if (typeof define === "function" && define.amd) {
        define("typeahead.js", [ "jquery" ], function(a0) {
            return factory(a0);
        });
    }
```
to 
```
 if (typeof define === "function" && define.amd) {
        define("typeahead", [ "jquery" ], function(a0) {
            return factory(a0);
        });
    }
```

after this change requireJS can correctly load the typeahead library. Please fix :-)

**Comment by yanickrochon** [comment]
I don't know why this hasn't been published, yet. The `dist` folder does contain the fix, but the bower module and other downloads date back to version 0.11.1.

**Comment by jcrben** [comment]
@yanickrochon It is indeed. We're not really doing a very good job - are you interested in helping out? I'm not sure the owner @lenovouser is still checking out the repo.

It looks like the npm package is handled by https://github.com/mariusz-kraj but he isn't a collaborator for this repo.

**Comment by yanickrochon** [comment]
@jcrben npm can handle such cases, where repo maintainers are no longer active, and the goal is to keep the community up-to-date modules. I personally have enough on my hands at the moment to be a maintainer of this project, however I could collaborate, sure, since I am using this widget. What are you suggesting?

**Comment by jlbooker** [comment]
FYI, I've initiated a package name dispute with NPM support. We're working on a new release in #55 .

**Comment by jlbooker** [comment]
I think was already fixed in 212fe89300e4760f15df4dec4e447d1dee21e965. The module is registered anomalously now, so the offending line in the Gruntfile has been completely removed. A newer build (from the newer Gruntfile) should fix this.
